code/modules/mob/living/carbon/human/species/grey.dm

```diff
@@ -88,3 +88,7 @@
 		H.adjustFireLoss(1)
 		return TRUE
 	return ..()
+
+/datum/species/grey/get_species_runechat_color(mob/living/carbon/human/H)
```

Comment:
Need to account for drask and kidan too.

Reply:
![image](https://user-images.githubusercontent.com/25063394/95628113-4f21d200-0a75-11eb-960e-662640403294.png)
![image](https://user-images.githubusercontent.com/25063394/95628116-50eb9580-0a75-11eb-9467-8fbb550385b1.png)